adserver/tasks.py

```diff
@@ -635,6 +635,9 @@ def update_previous_day_reports(day=None):
     daily_update_uplift(start_date)
     daily_update_regiontopic(start_date)
 
+    # Updates an aggregation on each paid flight
+    update_flight_traffic_fill.apply_async()
```

Comment:
Should we also prefix this with `daily` to match all the others? Also seems like the comment should say why this one is running another celery task instead of running it inline.

Reply:
Well, this task doesn't update "daily" numbers. It updates the lifetime of the flight. I'll add a comment about why it can be async though.